scripts/clean-go-files.ps1

```diff
@@ -2,6 +2,6 @@ $directories = Get-ChildItem -Path $env:MainDirectory -Directory -Exclude @("cor
 foreach ($directory in $directories) {
 	Remove-Item -Path $directory.FullName -Recurse -Force -Verbose
 }
-Remove-Item -Path $env:MainDirectory -Filter "*.go" -Exclude "graph_request_adapter.go" -Verbose
+Get-ChildItem -Path $env:MainDirectory -Filter graph_base_service_client.go -Recurse | Remove-Item
```

Comment:
Remove-Item -Path $env:MainDirectory -Filter "*.go" -Exclude "graph_request_adapter.go", "graph_service_client.go" -Verbose

Reply:
My only concern is that we will be introducing [tests](https://github.com/microsoftgraph/msgraph-sdk-go/blob/feat/simplify-client-initialization/graph_service_client_test.go) , as well as possibly some utility methods. i.e helper methods for the batch requests. Given that the generation only generates 1 file while all the other top level files  are manually written, wouldn't it be easier specify the file to delete rather than files to keep

Reply:
ok but the current script won't remove anything else but the generated client, not the models or the request builders. Which we want to remove for sanity. Maybe have an array defined above for the exclude values entries? We could also move the tests to a subdirectory I believe if that makes things easier.

Reply:
moved the tests to a separate folder